application/src/main/java/org/opentripplanner/datastore/api/FileType.java

```diff
@@ -15,7 +15,7 @@ public enum FileType {
   GTFS("🚌", "GTFS data"),
   NETEX("🚌", "NeTEx data"),
   EMISSION("🌿", "Emission data"),
-  EMPIRICAL_DATA("📊", "Emperical data"),
+  EMPIRICAL_DATA("📊", "Empirical data"),
```

Comment:
It's called empirical _data_, not delay. Do you expect more types in the future?